openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -461,6 +462,8 @@ def login_user(request):
     """
     _parse_analytics_param_for_course_id(request)
 
+    do_action("post_login", request, request.user)
```

Comment:
I think that a better place for this would be at the end of the login process, you could check tthe position in the other PR